packages/xrpl/test/models/Batch.test.ts

```
@@ -0,0 +1,136 @@
+      ValidationError,
+      'Batch: invalid field BatchSigners',
+    )
+    assert.throws(
+      () => validate(tx),
+      ValidationError,
+      'Batch: invalid field BatchSigners',
+    )
+  })
+
+  it('throws w/ missing RawTransactions', function () {
+    delete tx.RawTransactions
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)